ext/pdo/pdo.stub.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+function pdo_drivers(): array {}
+
+class PDO {
+    public function __construct(string dsn, string username = "", string passwd = "", array options = []) {}
+
+    public function beginTransaction(): bool {}
```

Comment:
IIRC we can't add parameters/return types for methods as they BC with the old signatures for those who extend PDO :confused:

Reply:
Ahh, I see! Thanks for pointing it out. I saw at the Date stubs that sometimes there are return type declarations, and sometimes only phpdoc hints, but I couldn't guess the reason.

Reply:
I discovered in the meanwhile that only return type declarations harm BC (my guess is that parameters are not a problem because of the partial contravariance introduced in PHP 7.2).